src/main/resources/assets/dungeons_arise/lang/pt_br.json

```diff
@@ -0,0 +1,32 @@
+{
+    "filled_map.dungeons_arise:monastery": "Mapa do Explorador do Mosteiro",
+    "filled_map.dungeons_arise:illager_campsite": "Mapa do Explorador do Acampamento Illager",
+    "filled_map.dungeons_arise:illager_castle": "Mapa do Explorador do Castelo Illager",
+    "filled_map.dungeons_arise:illager_hall": "Mapa do Explorador do Illager Hall",
+    "filled_map.dungeons_arise:illager_fort": "Mapa do Explorador do Forte Illager",
+    "filled_map.dungeons_arise:abandoned_temple": "Mapa do Explorador do Templo Abandonado",
+    "filled_map.dungeons_arise:lighthouse": "Mapa do Explorador do Farol",
+    "filled_map.dungeons_arise:mushroom_mines": "Mapa do Explorador de Minas de Cogumelo",
+    "filled_map.dungeons_arise:mushroom_village": "Mapa do Explorador da Aldeia do Cogumelo",
+    "filled_map.dungeons_arise:coliseum": "Mapa do Explorador do Coliseu",
+    "filled_map.dungeons_arise:fishing_hut": "Mapa do Explorador da Cabana de Pesca",
+    "filled_map.dungeons_arise:small_prairie_house": "Mapa do Explorador da Pequena Pradaria",
+    "filled_map.dungeons_arise:wishing_well": "Mapa do Explorador do Poço dos Desejos",
+    "filled_map.dungeons_arise:merchant_campsite": "Mapa do Explorador do Acampamento do Comerciante",
+    "filled_map.dungeons_arise:infested_temple": "Mapa do Explorador do Templo Infestado",
+    "filled_map.dungeons_arise:heavenly_rider": "Mapa do Explorador do Cavaleiro Celestial",
+    "filled_map.dungeons_arise:mining_system": "Mapa do Explorador do Sistema de Mineração",
+    "filled_map.dungeons_arise:heavenly_conqueror": "Mapa do Explorador do Conquistador Celestial",
+    "filled_map.dungeons_arise:scorched_mines": "Mapa das Minas Queimadas",
+    "filled_map.dungeons_arise:undead_pirate_ship": "Mapa do Explorador do Navio Pirata Morto-Vivo",
+    "filled_map.dungeons_arise:foundry": "Mapa do Explorador de Fundição",
+    "filled_map.dungeons_arise:small_blimp": "Mapa do Explorador do Dirigível Pequeno",
+    "filled_map.dungeons_arise:bandit_village": "Mapa do Explorador da Vila dos Bandidos",
+    "filled_map.dungeons_arise:typhon": "Mapa do Explorador do Typhon",
+    "filled_map.dungeons_arise:ceryneian_hind": "Mapa do Explorador Ceryneian Hind",
+    "filled_map.dungeons_arise:heavenly_challenger": "Mapa do Explorador do Desafiante Celestial",
+    "filled_map.dungeons_arise:illager_corsair": "Mapa do Explorador Illager Corsair",
+    "filled_map.dungeons_arise:illager_galley": "Mapa do Explorador da Galera Illager",

```

Comment:
I'm open to discuss this one, as "Galé" could also be the translation to "Galley", since "Galera" would look like we talking about the "Illager people" (as "galera" is an expression to a group of people in portuguese)